lib/Container/CoreExtension.php

```diff
@@ -219,6 +216,13 @@ private function registerComposer(Container $container)
                 $autoloaders[] = $autoloader;
             }
 
+            if ($autoloaders === []) {
```

Comment:
In this case I guess we should keep track of the autoloader paths we tried and list _all_ the ones we didn't find

Reply:
Potentially quite easy to test, might be nice to break the responsibility out of the CoreExtension.

Reply:
I've adjusted the exception to list all tried paths if nothing worked.

If you tell me how to restructure, so where to move the functionality, I'll try.

Reply:
What was the use case for this again? Currently if an autoloader is not existing, then an exception is thrown, which seems fine to me ... ?

Reply:
The use case was to configure all possible locations for autoloading in `~/.config/phpactor/phpactor.yml` and don't break if one does not exist.

This way you can configure all possible paths on your system, e.g. paths for TYPO3, NEOS, Flow, Drupal, which might all be different.

So you can provide ONE configuration to match them all.

Reply:
But with current `master` and this patch, I run into the following error in one of our project:
```
htdocs > ~/.vim/bundle/phpactor/bin/phpactor
PHP Fatal error:  Uncaught Symfony\Component\Debug\Exception\ClassNotFoundException: Attempted to load class "AddMissingProperties" from namespace "Phpactor\CodeTransform\Adapter\WorseReflection\Transformer".
Did you forget a "use" statement for another namespace? in /Users/siepmann/.dotfiles/.vim/bundle/phpactor/lib/Container/CodeTransformExtension.php:151
Stack trace:
#0 /Users/siepmann/.dotfiles/.vim/bundle/phpactor/vendor/phpbench/container/lib/Container.php(118): Phpactor\Container\CodeTransformExtension->Phpactor\Container\{closure}(Object(Phpactor\Container\ApplicationContainer))
#1 /Users/siepmann/.dotfiles/.vim/bundle/phpactor/lib/Container/CodeTransformExtension.php(126): PhpBench\DependencyInjection\Container->get('code_transform....')
#2 /Users/siepmann/.dotfiles/.vim/bundle/phpactor/vendor/phpbench/container/lib/Container.php(118): Phpactor\Container\CodeTransformExtension->Phpactor\Container\{closure}(Object(Phpactor\Container\ApplicationContainer))
#3 /Users/siepmann/.dotfiles/.vim/bundle/phpactor/lib/Container/ in /Users/siepmann/.dotfiles/.vim/bundle/phpactor/lib/Container/CodeTransformExtension.php on line 151

Fatal error: Uncaught Symfony\Component\Debug\Exception\ClassNotFoundException: Attempted to load class "AddMissingProperties" from namespace "Phpactor\CodeTransform\Adapter\WorseReflection\Transformer".
Did you forget a "use" statement for another namespace? in /Users/siepmann/.dotfiles/.vim/bundle/phpactor/lib/Container/CodeTransformExtension.php on line 151

Symfony\Component\Debug\Exception\ClassNotFoundException: Attempted to load class "AddMissingProperties" from namespace "Phpactor\CodeTransform\Adapter\WorseReflection\Transformer".
Did you forget a "use" statement for another namespace? in /Users/siepmann/.dotfiles/.vim/bundle/phpactor/lib/Container/CodeTransformExtension.php on line 151

Call Stack:
    0.1231    6157632   1. Symfony\Component\Debug\ErrorHandler->handleException() /Users/siepmann/.dotfiles/.vim/bundle/phpactor/vendor/symfony/debug/ErrorHandler.php:0
```